site/content/en/docs/Guides/Client SDKs/_index.md

```diff
@@ -316,6 +316,96 @@ or SDK.WatchGameServer() instead to list the connected players.
 
 [playerstatus]: {{< ref "/docs/Reference/agones_crd_api_reference.html#agones.dev/v1.PlayerStatus" >}}
 
+### Counters And Lists
+
+{{< alpha title="Counters And Lists" gate="CountersAndLists" >}}
+
+The `Counters` and `Lists` features in the SDK offer a flexible configuration for tracking various entities like 
+players, rooms, and sessions. However, it's important to note that this feature has been in Alpha stage and disabled
+by default.
+
+### Counters
+
+#### Alpha().GetCounterCount(key)
+
+This function retrieves the current count for a specified Counter in the game server. It ensures the count is accurate
+based on the predefined keys in the GameServer resource, and returns an error if the key is not predefined.
+
+#### Alpha().SetCounterCount(key, amount)
+
+This function sets the count to a given value. This operation overwrites any previous values and the new value cannot
+exceed the Counter's capacity. The function returns `true` if successful, along with an `error` if the operation fails.
+
+#### Alpha().IncrementCounter(key, amount)
+
+This function increments the count of a specified Counter by a given nonnegative amount. It updates the current CRD 
+value and maxes out at the maximum value of `int64`. The function returns an `error` if the key wasn't predefined in 
+the GameServer resource and returns `false` if the Counter is already at capacity, indicating no increment will occur.
+There is a potential race condition when count values are set from both the SDK and through the K8s API(Allocation or otherwise),
```

Comment:
We should probably move this comment up to the top (under the Counters and Lists title), as variation of this comment applies to any of the update operations.

Reply:
Moved this line under the title.